atom/browser/native_window_views_win.cc

```diff
@@ -92,15 +92,25 @@ bool NativeWindowViews::PreHandleMSG(
     // accessibility object.
     case WM_GETOBJECT: {
       const DWORD obj_id = static_cast<DWORD>(l_param);
-      if (enabled_a11y_support_) return false;
-
-      if (obj_id == OBJID_CLIENT) {
-        const auto axState = content::BrowserAccessibilityState::GetInstance();
-        if (axState && !axState->IsAccessibleBrowser()) {
-          axState->OnScreenReaderDetected();
-          enabled_a11y_support_ = true;
-          Browser::Get()->OnAccessibilitySupportChanged();
-        }
+      if (checked_for_a11y_support_) return false;
+
+      checked_for_a11y_support_ = true;
```

Comment:
We can do that, I was trying to make this as efficient as possible but I didn't know we had an event for this, :+1: